src/routes/About/About.js

```diff
@@ -0,0 +1,7 @@
+export default function About(){
+    return(
+        <div>
```

Comment:
React supports `<>...</>` empty elements when you just need a container for a fragment. These don't result in any output so avoid cluttering your dom
```suggestion
        <>
```